### ISSUE TITLE
Revaccination

### ISSUE THREAD

**Issue by giovannic** [mrc-ide/hypatia#22]
Contingent on #20 

Vaccinating an individual with a vaccine different to their initial vaccine.

 - [ ] Model vaccine response based on latest vaccination (first iteration)
 - [ ] Implement correlation between vaccines taken (need to confirm a model still)

**Comment by pwinskill** [comment]
We are defining re-vaccination as vaccination with a new vaccine type following an original vaccination.

We will need to decide how impacts of multiple vaccines (with potentially different characteristics) interact. To date, there is no information (for COVID19) to parameterise these interactions and the best approach may be to get expert opionion from immunologists.

**Comment by abhogan** [comment]
The simplest approach might be to assume that the individual is protected according to the characteristics of the most recent vaccine that they received.

We will also need to decide how to deal with correlation - i.e. individuals who have been vaccinated previously are more likely to get vaccinated again, and some people will never get any vaccine.

**Comment by pwinskill** [comment]
Good point on correlation. I've just reviewed Giovanni's correlation implementation for the malaria model which we could use for inspiration if needed?
